tools/releaseBuild/build.json

```diff
@@ -69,7 +69,7 @@
         ],
         "DockerImageName": "ps-winsrvcore",
         "BinaryBucket": "symbols",
```

Comment:
if we change the containerfolder in the other PR, we need to change the binarybucker where it equals symbols to `results`

Reply:
Yes, the other PR has been updated. I also changed `BinaryBucket` to `results` for the `-symbols` entries.